pytest.ini

```diff
@@ -11,3 +11,6 @@ filterwarnings =
     ignore:.*The NumPy module was reloaded*:UserWarning
 doctest_optionflags = NUMBER ELLIPSIS
 testpaths = tests
+markers =
+    needs_vtk9: skip test unless VTK is 9.0 or newer.
+    needs_vtk_version(version): skip test unless VTK version is at least as specified.
```

Comment:
Great idea! Only one point I am wondering is we need to split `nees_vtk9`. In my mind we can express it with `needs_vtk_version(9.0.0)`

Reply:
That would certainly work. The only reason why I defined a specific marker is that we have a lot of those cases in the test suite, and this way we don't have to define a `skip_if_vtk9` at the top of a bunch of test modules. After all the point of this PR is to reduce code duplication.

Major version bumps will always introduce a lot of new things. I expect we'll need `needs_vtk10` eventually. So we also can't default `needs_vtk_version` to 9.